packages/common/src/search/_internal/hubEventsHelpers/processFilters.ts

```diff
@@ -15,10 +15,24 @@ export function processFilters(filters: IFilter[]): Partial<GetEventsParams> {
   const processedFilters: Partial<GetEventsParams> = {};
   const access = getOptionalPredicateStringsByKey(filters, "access");
   if (access?.length) {
-    // TODO: remove ts-ignore once GetEventsParams supports filtering by access
-    // @ts-ignore
     processedFilters.access = access;
   }
+  const canEdit = getOptionalPredicateStringsByKey(filters, "canEdit");
```

Comment:
Maybe we should expand the comment/docs for `hubSearchEvents` to concretely enumerate all the supported filter parameter values for searching for events.

```ts
/**
 * Searches for events against the Events 3 API using the given `query` and `options`.
 * Currently supported filters include:
 *   access: boolean
 *   canEdit: boolean
 *   entityIds: string;
 *   entityTypes: string;
 *   eventIds: string;
 *   term: string;
 *   categories: string;
 *   tags: string;
 *   readGroupIds: string;
 *   editGroupIds: string;
 *   attendanceType: string;
 *   owner: string;
 *   status: string;
 *   startDateRange: IDateRange<string | number>
 * @param query An IQuery object
 * @param options An IHubSearchOptions object
 * @returns a promise that resolves a <IHubSearchResponse<IHubSearchResult> object
 */
export async function hubSearchEvents(
  // ...
}
```

Reply:
You replaced the existing comment for `processFilters` with the one I requested above, but it should have been added to the `hubSearchEvents` method. Lets copy this comment to `hubSearchEvents` and revert the comment/doc change for `processFilters`.

Reply:
Also, I realize now that I had the `access` filter typed as `boolean` when it should be `string`.